app/models/document.rb

```diff
@@ -39,7 +39,7 @@ def temporary_update_type
   alias_method :temporary_update_type?, :temporary_update_type
 
   validates :title, presence: true
-  validates :summary, presence: true
+  validates :summary, presence: true, unless: -> { protected_food_drink_name? }
```

Comment:
The syntax can be simplified slightly, like the examples below:

```suggestion
  validates :summary, presence: true, unless: :protected_food_drink_name?
```

Reply:
Ah, good point! 😉